Formula/nim.rb

```diff
@@ -16,24 +16,28 @@ class Nim < Formula
   def install
     if build.head?
       system "/bin/sh", "bootstrap.sh"
+
+      # Grab the tools source (only included in point releases)
+      system "git", "clone", "https://github.com/nim-lang/nimsuggest.git", "dist/nimsuggest"
+      system "git", "clone", "https://github.com/nim-lang/nimble.git", "dist/nimble"
     else
       system "/bin/sh", "build.sh"
     end
     system "/bin/sh", "install.sh", prefix
 
-    system "bin/nim e install_tools.nims"
+    bin.install_symlink prefix/"nim/bin/nim"
+    bin.install_symlink prefix/"nim/bin/nim" => "nimrod"
 
+    # build and install the tools
+    system "bin/nim", "e", "install_tools.nims"
```

Comment:
Is there a longer option that can be used instead of `e`? Can remove the comment, it's self-explanatory from the code.